open-api/swagger.json

```diff
@@ -1566,6 +1566,19 @@
         }
       }
     },
+    "/v2/tree_nodes/clear_and_update": {
+      "get": {
+        "description": "This endpoint should be called after loading, deleting or updating data in the database. Clears tree nodes, counts caches, patient sets and bitsets. Updates data for subscribed user queries.\nOnly for administrators.\n",
+        "tags": [
+          "v2"
+        ],
+        "responses": {
+          "200": {
+            "description": "The cache and patient sets have been cleared. The data for subscribed user queries is updated.\n"
+          }
+        }
+      }
+    },
```

Comment:
This call is an extension for "/v2/tree_nodes/clear_cache", but it is not a best fit for the /v2/tree_nodes call. @gijskant @forus do you have some suggestions on how to fit it in the API?

Reply:
You could create a separate controller, maybe a SystemController or MaintainanceController and have something like a /v2/system/clear_cache, /v2/maintainaince/dataloadingFinished, something descriptive and concise...